lib/bundler.rb

```diff
@@ -451,6 +451,10 @@ def system_bindir
       Bundler.settings[:system_bindir] || Bundler.rubygems.gem_bindir
     end
 
+    def gemfile
```

Comment:
There's a `Bundler.default_gemfile` method which returns the full path to the default `gems.rb` or `Gemfile` file, so I think introducing this method could create confusion.

Maybe a `Bundler.settings.preferred_gems_file_name` method instead?

```ruby
def preferred_gems_file_name
  self[:init_gems_rb] ? "gems.rb" : "Gemfile"
end
```